main.tf

```diff
@@ -159,13 +200,14 @@ module "tfe_workspace" {
   terraform_version              = var.tfe_workspace.terraform_version
   trigger_prefixes               = var.tfe_workspace.connect_vcs_repo != false ? var.tfe_workspace.trigger_prefixes : null
   username                       = var.tfe_workspace.username
+  variable_set_ids               = { "account" : tfe_variable_set.account.id }
```

Comment:
you need to merge this with var.tfe_workspace.variable_set_ids

Reply:
Indeed, fixed.